chromium/pages/popup/style.css

```diff
@@ -1,128 +1,316 @@
-html {
-  background-color: #fff
+:root {
+  --https-blue: #0a84ff;
+  --darker-blue: #0060df;
+  --text-main: #000;
+  --text-secondary: #464646;
+  --ligh-grey: #ececec;
```

Comment:
light-grey?

Reply:
Are you asking about why it's called that? Not sure what the question is here.

Reply:
@zoracon They probably think it's a typo, which it indeed looks like.

Reply:
Oh, I see now. Thanks for catching that